docs/data/sources/google-tag-manager/index.md

```diff
@@ -25,9 +25,7 @@ Amplitude GTM templates are custom templates which define tags and variables so
 
 Google tags are snippets of code that measure user activity such as time on page, clicks, and purchases.
 
-!!! warning "Messages are queued"
-
-    To enforce the processing order of messages pushed to the data layer, add an event name to a message as it is pushed to the data layer, and then listen for that event name with a Custom Event trigger. Learn more about [how Google processes data layer information](https://developers.google.com/tag-platform/devguides/datalayer?hl=en#how_data_layer_information_is_processed).
+--8<-- "includes/gtm/data-layer-messages-warning.md"
```

Comment:
Should we remove this from here?
I feel this page is more generate to all type of container.

Reply:
I think that makes sense.

I was also thinking we could add links in each section. wdyt?

<img width="731" alt="image" src="https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/assets/5790872/149b464c-6ad4-40a9-abd2-a0539e8b2b47">